src/population/visualization/graphing_model.py

```diff
@@ -0,0 +1,205 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+from torch.utils.data import DataLoader, TensorDataset
+import numpy as np
+
+
+class GraphingModel(nn.Module):
+    """
+    A simple neural network used to perform a mapping of N to 2,
+    where N is the maximum number of genes.
+    The 2D result is used for visualization purposes.
```

Comment:
Line length according to our formatter is like 120 characters, I would recommend using them all.

Reply:
Kinda hard to read on my screen if the lines are too long.